modules/network-annotation-processor/src/integrationTest/java/org/apache/ignite/internal/network/processor/ItTransferableObjectProcessorTest.java

```diff
@@ -170,7 +170,8 @@ void testMissingMessageGroup() {
         Compilation compilation = compiler.compile(sources("AllTypesMessage"));
 
         assertThat(compilation).hadErrorContaining(
-                "No message groups (classes annotated with @MessageGroup) found"
+                "No message groups (classes annotated with @MessageGroup) found while processing messages from the "
```

Comment:
Maybe use: "@" + MessageGroup.class.getSimpleName()
In case the name of the annotation changes.